### DISCUSSION TITLE
(Solved) upgraded to 1.2.0 and all my plots no longer showing up

### DISCUSSION THREAD

**Discussion by ZielReaver** [Chia-Network/chia-blockchain#7185] [Support]
Hello I am in need of major help. I just updated to the official 1.2.0 release and now under plots none of my plots are showing up and removed directories and then them up again but none of my over 1700 plots are showing up any more.

**Comment by mariano54** [comment]
Wait a few minutes and click the refresh button in the plots page

**Comment by ZielReaver** [comment]
Yep that fixed it but now I am seeing it looks like the GUI is not auto refreshing anymore. When I woke up this morning I had to hit the fresh button on all three of my systems.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
